### PR TITLE
fix: hopper conflict and reload-command

### PR DESCRIPTION
**Pull request by Sh1roCu**
fix hopper conflict when shift+rightclick;
fix reload-command can't reload config correctly.

### CODE REVIEW

File: src/main/java/me/entity303/openshulker/commands/OpenShulkerCommand.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command
         }
 
         if (args[0].equalsIgnoreCase("reload")) {
-            this._openShulker.reloadConfig();
+            this._openShulker.reloadCfg();
```
Comment (Test-Account666) [inline src/main/java/me/entity303/openshulker/commands/OpenShulkerCommand.java:37]: I don't see a point in creating the reloadCfg method.

Just publicize `InitializeConfig` and execute that.

**Review by Test-Account666 — CHANGES_REQUESTED** [review]
Thanks for the Pull Request btw :)

**Review by Sh1roCu — COMMENTED** [review]
done

**Review by Test-Account666 — APPROVED** [review]
Looks fine to me now. Thanks for your contribution. I really appreciate it :)

**Pull request merged**